nav2_behaviors/CMakeLists.txt

```diff
@@ -0,0 +1,80 @@
+cmake_minimum_required(VERSION 3.5)
+project(nav2_behaviors)
```

Comment:
I would rename this to `nav2_recovery_behaviors` unless you're intending to support other non-recovery behaviors here

Reply:
This will host all sorts of behaviors, used for recovery or some other non-recovery task, i.e. backing up could be used as a recovery action or for docking the robot.

Reply:
The ROS-y way of doing this isnt to make 1 large package but many smaller packages. I would encourage us to make a distinction between recovery and social/other behaviors that are event, opportunistic based and not as a result of planning/control failure. 

With that said, I generally eye larger packages with many themed executables, but the point stands that I think a distinction between recoveries and social behaviors is necessary as the way they're used and how they're to be triggered will differ dramatically.